### PR TITLE
feat(cluster-scanner): Add dynamic envs to values

### PR DESCRIPTION
**Pull request by FedericoFeresini**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/cluster-scanner/templates/deployment.yaml [hunk 1]
```diff
@@ -277,6 +277,14 @@ spec:
...
                 key: rsi_js_server_metrics_port
                 optional: true
+          {{- range $key, $value := .Values.runtimeStatusIntegrator.env }}
+          - name: "{{ $key }}"
+            value: "{{ $value }}"
+          {{- end }}
+          {{- range $key, $value := .Values.imageSbomExtractor.env }}
+          - name: "{{ $key }}"
+            value: "{{ $value }}"
+          {{- end }}
```
Comment (ProvoK) [inline charts/cluster-scanner/templates/deployment.yaml]: shouldn't this be in the `ise` container below?

File: charts/cluster-scanner/values.yaml [hunk 2]
```diff
@@ -147,6 +147,8 @@ runtimeStatusIntegrator:
       # Runtime Status Integrator Memory requests per replica
       memory: 350Mi
 
+  env: {}
```
Comment (ProvoK) [inline charts/cluster-scanner/values.yaml:151]: I know it's a bit of a PITA, but could we add a couple of unittest? 🙏

**Review by ProvoK — COMMENTED** [review]
left a couple of comments, thanks for working on this @FedericoFeresini

File: charts/cluster-scanner/values.yaml (hunk 2 above)
Comment (michele-mangili) [reply charts/cluster-scanner/values.yaml:151]: I agree, we shall add them!

File: charts/cluster-scanner/templates/deployment.yaml (hunk 1 above)
Comment (FedericoFeresini) [reply charts/cluster-scanner/templates/deployment.yaml]: Thanks a lot  @ProvoK, I had misplaced it

**Review by michele-mangili — APPROVED** [review]

**Review by ProvoK — APPROVED** [review]
LGTM thanks @FedericoFeresini !

**Pull request merged**